### DISCUSSION TITLE
sinkhorn limit

### DISCUSSION THREAD

**Discussion by nycos62** [josdejong/mathjs#3268] [Ideas]
Hello, 
should we consider to implement sinkhorn limit for matrices in the framework ?

see: https://www.youtube.com/watch?v=-uIwboK4nwE

**Comment by josdejong** [comment]
Thanks for your suggestion. If there is enough interest in this we can consider it. I have the feeling though that his is quite a niche. You can always implement this as a standalone plugin or function that can be imported in mathjs straightaway.
